src/CGIHandler.cpp

```diff
@@ -157,7 +157,7 @@ bool CGIHandler::executeCGI(const MetaVariables &env, HTTPResponse &response)
 		// execve(argvPointers[0], argvPointers.data(), envpPointers.data());
 		if (execve(argvPointers[0], argvPointers.data(), envpPointers.data()) == -1)
 		{
-			perror("execve");
+			Debug::log("CGIHandler: execve failed", Debug::CGI);
 			return false;
```

Comment:
perror gives extra information and this way we are losing them. Please provide a Debug::log version that integrates the perror functionalities.

Reply:
Alright